README.md

```diff
@@ -133,19 +166,31 @@ NOTE:  The last rule means that expressions such as
 
 produce early errors because of recursive application of the first rule.
 
+
+### Bound Constructor Wrapper Functions ###
+
+A bound constructor wrapper function is an anonymous built-in function that has a [[BoundTargetConstructor]] internal slot.
+
+When a bound constructor wrapper function _F_ is called with zero or more _args_, it performs the following steps:
+
+- Assert: _F_ has a [[BoundTargetConstructor]] internal slot.
+- Let _target_ be the value of _F_'s [[BoundTargetConstructor]] internal slot.
+- Assert: IsConstructor(_target_).
+- Let _args_ be a List consisting of all the arguments passed to this function.
+- Let _value_ be ? Construct(_target_, _args_).
+- Return _value_.
+
+Note: Bound constructor wrapper functions are never directly accessible to ECMAScript code.
+
+
 ### Runtime Semantics ###
 
-    BindExpression :
-        LeftHandSideExpression :: [lookahead ≠ new] MemberExpression
+**InitializeBoundFunctionProperties(bound, target)**
```

Comment:
_F_ is used as the parameter name in the steps, so we need to either change those references to _bound_ or change the parameter name here.